data/translations/pt_PT.ts

```diff
@@ -2,6 +2,20 @@
 <!DOCTYPE TS>
 <TS version="2.1" language="pt_PT">
 <context>
+    <name>PictureBox</name>
+    <message>
+        <source>Press to login</source>
+        <translation type="unfinished"></translation>
```

Comment:
@plfiorini, I also fixed Ukrainian translation and added some translation files to CMakeFiles (yes, they were committed without change in CMakeFiles…)